test/SILGen/accessibility_warnings.swift

```diff
@@ -141,19 +141,19 @@ internal class InternalClass {
 }
 
 package class PackageClass {
-  // CHECK-DAG: sil{{( \[.+\])*}} [ossa] @$s22accessibility_warnings12PackageClassC9publicVarSivg
+  // CHECK-DAG: sil package [transparent] [ossa] @$s22accessibility_warnings12PackageClassC9publicVarSivg
   public var publicVar = 0
```

Comment:
1. add library evolution - getter shouldn't be transparent 
2. compare with public